com.unity.render-pipelines.high-definition/Editor/Material/Lit/ShaderGraph/HDLitSubTarget.cs

```diff
@@ -41,7 +41,6 @@ public HDLitData litData
         };
 
         protected override string[] templateMaterialDirectories => passTemplateMaterialDirectories;
-        protected override string customInspector => "Rendering.HighDefinition.LitShaderGraphGUI";
```

Comment:
be carefult, this one is protected override string customInspector => "Rendering.HighDefinition.LitShaderGraphGUI"; and it isn't "Rendering.HighDefinition.LightingShaderGraphGUI"; .  
Is it equivalent? does it change something? I guess it should have been lighting, but just checking

Reply:
Indeed, it should be LitShaderGraphGUI and not the lighting one. Thanks!